starboard/shared/ffmpeg/ffmpeg_dispatch.h

```diff
@@ -48,7 +48,12 @@ constexpr int kAVUtilSupportsBufferCreate = 3409920;
 // https://github.com/FFmpeg/FFmpeg/blob/70d25268c21cbee5f08304da95be1f647c630c15/doc/APIchanges#L195
 // avcodec_decode_audio4 and avcodec_decode_video2 replaced by
 // avcodec_receive_frame()
-constexpr int kAVCodecHasUniformDecodeAPI = 3940198;
+//
+// The APIs were removed in this change:
+// https://github.com/FFmpeg/FFmpeg/commit/7c1f347b184b6738abdc22fdcda40baa9f932522#diff-76418b674d0db8d5027d2e1e325dbe9b92b65b09d9f20cdd305ad14b0e46562d
+// (note the values in libavcodec/version.h)
+// AV_VERSION_INT(58, 137, 100)
```

Comment:
Here I just picked the libavcodec version from the change that removes avcodec_decode_audio4 and avcodec_decode_video2. Let me know if a different version makes more sense (I couldn't figure out which one 3940198 was referring to).